redbot/setup.py

```diff
@@ -16,6 +16,8 @@
 from redbot.core.data_manager import basic_config_default
 from redbot.core.json_io import JsonIO
 from redbot.core.utils import safe_delete
+from redbot.core.drivers.red_mongo import Mongo, get_config_details
```

Comment:
You can't import these here as the bot will fail if the user doesn't have `motor` installed. Both of these imports should be in the `X_to_Y` functions.